ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -358,6 +362,44 @@ def ocs_install_verification(
         # a verification of the installation of it will run
         # on all running state pods
         check_fips_enabled()
+    if config.ENV_DATA.get("encryption_at_rest"):
+        osd_encryption_verification()
+
+
+def osd_encryption_verification():
+    """
+    Verify if OSD encryption at rest if successfully deployed on OCS
+
+    :raise: UnsupportedFeatureError, EnvironmentError
```

Comment:
```suggestion
    Raises:
      UnsupportedFeatureError: when
      EnvironmentError: when
```

Look at example here:
https://github.com/red-hat-storage/ocs-ci/blob/dc944d2de87bdde0dd7a39c1bda2843f1ea9feb4/ocs_ci/ocs/resources/packagemanifest.py#L45